### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ZeJ0hn**
The current code doesn't compile.
Need to use currentTarget instead of target (because target is generic now)

### CODE REVIEW

**Comment by ZeJ0hn** [comment]
I signed it!

File: packages/drawer/README.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ export default class App extends Component {
       <div className='drawer-container'>
         <TopAppBar title='Inbox'/>
 
-        <TopAppBarFixedAdjust>
+        <TopAppBarFixedAdjust className='top-app-bar-fix-adjust'>
```
Comment (moog16) [inline packages/drawer/README.md]: revert everything in this file since you have it in #666

**Comment by moog16** [comment]
please update

**Comment by ZeJ0hn** [comment]
made a mistake. 
push again with #670

**Pull request closed without merging**